posthog/session_recordings/test/test_session_recordings.py

```diff
@@ -543,7 +575,7 @@ def test_get_snapshots_v2_default_response(self, mock_list_objects: MagicMock, _
             f"session_recordings/team_id/{self.team.pk}/session_id/{session_id}/data/{timestamp - 10000}-{timestamp - 5000}",
             f"session_recordings/team_id/{self.team.pk}/session_id/{session_id}/data/{timestamp - 5000}-{timestamp}",
         ]
-        response = self.client.get(f"/api/projects/{self.team.id}/session_recordings/{session_id}/snapshots?version=2")
+        response = self.client.get(f"/api/projects/{self.team.id}/session_recordings/{session_id}/snapshots?")
```

Comment:
```suggestion
        response = self.client.get(f"/api/projects/{self.team.id}/session_recordings/{session_id}/snapshots")
```

if theres no query params no need to add the `?`